python/tests/behaviour/connection/user/user_steps.py

```diff
@@ -15,114 +15,64 @@
-#         pass
-#
-#
-#
-# @step("users not contains: {username}")
-# def step_impl(context: Context, username: str):
-#     assert_that([u.username() for u in _get_driver(context).users.all()], not_(has_item(username)))
-#
-#
-# @step("users create: {username:NonSemicolon}, {password:NonSemicolon}")
-# def step_impl(context: Context, username: str, password: str):
-#     _get_driver(context).users.create(username, password)
-#
-#
-#
-# @step("users create: {username:NonSemicolon}, {password:NonSemicolon}; throws exception")
-# def step_impl(context: Context, username: str, password: str):
-#     try :
-#         _get_driver(context).users.create(username, password)
-#         assert False
-#     except TypeDBDriverException:
-#         pass
-#
-#
-# @step("users get all")
-# def step_impl(context: Context):
-#     _get_driver(context).users.all()
-#
-#
-# @step("users get all; throws exception")
-# def step_impl(context: Context):
-#     try:
-#         _get_driver(context).users.all()
-#         assert False
-#     except TypeDBDriverException:
-#         pass
-#
-#
-# @step("users get user: {username:NonSemicolon}")
-# def step_impl(context: Context, username: str):
-#     _get_driver(context).users.get(username)
-#
-#
-# @step("users get user: {username:NonSemicolon}; throws exception")
-# def step_impl(context: Context, username: str):
-#     try :
-#         _get_driver(context).users.get(username)
-#         assert False
-#     except TypeDBDriverException:
-#         pass
-#
-#
-# @step("users delete: {username:NonSemicolon}")
-# def step_impl(context: Context, username: str):
-#     _get_driver(context).users.delete(username)
-#
-#
-#
-# @step("users delete: {username:NonSemicolon}; throws exception")
-# def step_impl(context: Context, username: str):
-#     try:
-#         _get_driver(context).users.delete(username)
-#         assert False
-#     except TypeDBDriverException:
-#         pass
-#
-#
-# @step("users password set: {username:NonSemicolon}, {password:NonSemicolon}")
-# def step_impl(context: Context, username: str, password: str):
-#     _get_driver(context).users.password_set(username, password)
-#
-#
-# @step("users password set: {username:NonSemicolon}, {password:NonSemicolon}; throws exception")
-# def step_impl(context: Context, username: str, password: str):
-#     try:
-#         _get_driver(context).users.password_set(username, password)
-#         assert False
-#     except TypeDBDriverException:
-#         pass
-#
-#
-# @step("users password update: {username}, {password_old}, {password_new}")
-# def step_impl(context: Context, username: str, password_old: str, password_new: str):
-#     _get_driver(context).users.get(username).password_update(password_old, password_new)
+
+@step("get all users")
+def step_impl(context: Context):
+    assert_collections_equal([db.name for db in context.driver.users.all()], parse_list(context.table))
+
+
+@step("get all users{may_error:MayError}")
```

Comment:
This may conflict with "get all users" above, because Python's cucumber parser is one of the not smartest ones. It was ok in my tests, but take into account if you face issues when I'm away